### DISCUSSION TITLE
Automate the close of issue related when PR is merged

### DISCUSSION THREAD

**Discussion by sabderemane** [djangopackages/djangopackages#727] [Feature Ideas]
**Is your feature request related to a problem? Please describe.**
Linked to [my comment](https://github.com/djangopackages/djangopackages/issues/688#issuecomment-931632949), I think it can be useful to automate the close of a related issue of a PR since maintainers hasn't always time to do so, or forget about it.

**Describe the solution you'd like**
Create an action on pull request events which close the issue linked to merged PRs.

**Additional context**
I've seen [this action](https://github.com/ldez/gha-mjolnir) which can be nice to do that.

**Comment by sabderemane** [comment]
cc @jefftriplett,

I have checked, there isn't so many changes between the fact of adding the "Fixes #nb-issue" (GitHub bot) and the action I mentioned :

> Useful:
> to close multiple issues related to a pull request.
> to close issues related to a pull request not based on the default branch (i.e. master). For example, when a branch is related to version (e.g. v1.5, v2.0, ...)

I'm not sure it's really needed in this project. Feel free to close the issue if think the same.

**Comment by jefftriplett** [comment]
@sabderemane thanks for the recommendation and for doing the research here. It looks useful and might be nice for lining up tickets for release better than my merge and deploy workflow. 

I'll clean up tickets and then convert this to discussion for follow up (to avoid #hacktoberfest burying it)
